pom.xml

```diff
@@ -153,6 +153,30 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>3.1.0</version>
```

Comment:
In Spring pom <maven-enforcer-plugin.version>3.0.0</maven-enforcer-plugin.version>